src/main/kotlin/game/racing/domain/Car.kt

```diff
@@ -0,0 +1,28 @@
+package game.racing.domain
+
+class Car {
+    var position = MIN_POSITION
```

Comment:
객체의 생성은 주생성자를 통해서 프로퍼티를 초기화해보면 어떨까요? 😃


```suggestion
class Car(var position = MIN_POSITION) {
```